src/Tizen.NUI.Components/Controls/Toast.cs

```diff
@@ -79,13 +100,21 @@ public static Toast FromText(string text, uint duration)
         private TextLabel textLabel = null;
         private string strText = null;
         private Timer timer = null;
-        private readonly uint duration = 3000;
+        private uint duration = 3000;
+        private Extents textPadding;
 
         /// <summary>
         /// Get style of toast.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
-        public new ToastStyle Style => ViewStyle as ToastStyle;
+        public new ToastStyle Style
```

Comment:
Hello,
this is only getter.
I think some explanation needs to be added how user can set style.

Reply:
I  will add the comment